app/javascript/src/components/Common/Menubar/index.jsx

```diff
@@ -4,62 +4,78 @@ import { Settings, Plus, Search } from "@bigbinary/neeto-icons";
+          <MenuBar.Block label="Archived" count={2} />
+          <MenuBar.Block label="Completed" count={7} />
+          <MenuBar.Block label="Phase 2" count={4} />
+        </>
+      ) : (
+        <>
+          <MenuBar.Block label="All" count={13} active />
+          <MenuBar.Block label="Users" count={2} />
+          <MenuBar.Block label="Leads" count={7} />
+          <MenuBar.Block label="Visitors" count={4} />
+        </>
+      )}
 
-        <MenuBar.SubTitle
-          iconProps={[
-            {
-              icon: Search,
-              onClick: () => setIsSearchCollapsed(!isSearchCollapsed),
-            },
-          ]}
+      <MenuBar.SubTitle
+        iconProps={[
+          {
+            icon: Search,
+          },
+        ]}
+      >
+        <Typography
+          component="h4"
+          style="h5"
+          textTransform="uppercase"
+          weight="bold"
         >
-          <Typography
-            component="h4"
-            style="h5"
-            textTransform="uppercase"
-            weight="bold"
-          >
-            Segments
-          </Typography>
-        </MenuBar.SubTitle>
-        <MenuBar.Block label="Europe" count={80} />
-        <MenuBar.Block label="Middle-East" count={60} />
-        <MenuBar.Block label="Asia" count={60} />
-        <MenuBar.SubTitle
-          iconProps={[
-            {
-              icon: Settings,
-            },
-            {
-              icon: Plus,
-            },
-            {
-              icon: Search,
-            },
-          ]}
+          Segments
+        </Typography>
+      </MenuBar.SubTitle>
+      {!isContactsPage && (
+        <>
+          <MenuBar.Block label="Europe" count={80} />
+          <MenuBar.Block label="Middle-East" count={60} />
+          <MenuBar.Block label="Asia" count={60} />
+        </>
+      )}
+      <MenuBar.SubTitle
+        iconProps={[
+          {
+            icon: Settings,
+          },
+          {
+            icon: Plus,
+          },
+          {
+            icon: Search,
+          },
+        ]}
+      >
+        <Typography
+          component="h4"
+          style="h5"
+          textTransform="uppercase"
+          weight="bold"
         >
-          <Typography
-            component="h4"
-            style="h5"
-            textTransform="uppercase"
-            weight="bold"
-          >
-            Tags
-          </Typography>
-        </MenuBar.SubTitle>
-        <MenuBar.Block label="Sales" count={80} />
-        <MenuBar.Block label="Finance" count={60} />
-        <MenuBar.Block label="User Experience" count={60} />
-      </MenuBar>
-    </div>
+          Tags
+        </Typography>
+      </MenuBar.SubTitle>
+      {!isContactsPage && (
+        <>
+          <MenuBar.Block label="Sales" count={80} />
+          <MenuBar.Block label="Finance" count={60} />
+          <MenuBar.Block label="User Experience" count={60} />
+        </>
+      )}
```

Comment:
Rather than doing conditional rendering here. Can we use a prop to switch between two constants each of which is for either `Contacts` or `Notes`.